DocumentDB/input.tf

```diff
@@ -0,0 +1,139 @@
+#########################################################################
+### DocumentDB Input Variables
+#########################################################################
+
+variable "master_username" {
+  description = "(Required) The username of the documentdb cluster."
+  sensitive   = true
+  type        = string
+}
+
+variable "master_password" {
+  description = "(Required). The password of the documentdb cluster."
+  sensitive   = true
+  type        = string
+}
+
+variable "database_name" {
+  description = "(Required). Name of the database."
+  type        = string
+}
+
+variable "cluster_family" {
+  type        = string
+  default     = "docdb5.0"
+  description = "(Required, default is `docdb5.0`). The family of the DocumentDB cluster parameter group. More information can be found at https://docs.aws.amazon.com/documentdb/latest/developerguide/cluster_parameter_groups.html"
+}
+
+variable "parameters" {
+  type = list(object({
+    apply_method = optional(string)
+    name         = string
+    value        = string
+  }))
+  default     = []
+  description = "(Optional). List of parameters to apply to the DocumentDB database"
+}
+
+variable "backup_retention_period" {
+  type        = string
+  default     = "7"
+  description = "(Optional, default is 7). The number of days that you should keep backups for."
+}
+
+variable "backup_window" {
+  type        = string
+  default     = "06:00-08:00"
+  description = "(Optional, default is `06:00-08:00`). Daily time range that backups execute (UTC time). Default is at 1:00am - 3:00am EST."
+}
+
+variable "apply_immediately" {
+  type        = bool
+  description = "(Optional, default is `true`). Determines if any cluster modifications are applied immediately, or during the next maintenance window. Note that apply immediately can result in a brief downtime as the cluster reboots."
+  default     = true
+}
+
+variable "deletion_protection" {
+  type        = bool
+  default     = true
+  description = "(Optional, default is `true`). Prevents the cluster from being deleted and indicates if deletion protection is enabled."
+}
+
+variable "storage_encrypted" {
+  type        = bool
+  description = "(Optional, default is `false`). Determines if the DB cluster is encrypted."
+  default     = false
```

Comment:
Could we default this to `true` just for more increased security or does it make things awkward setting up the module?

Reply:
@patheard - We could turn it on, but that means that we always need to have a kms key for encryption and we have to always encrypt it. Also, once it is turned on, encryption can't be disabled and you have to use the same kms key to restore a snapshot to another account if needed. I am ok to turn it on by default, but what do you think given the above?

Reply:
Gotcha, as a middle ground what would you think of not defaulting this value and forcing the user to enter `true/false` and think consider the consequences for their use-case.

I just don't want to accidentally give people the impression that their cluster is encrypted at rest by default like so many other AWS storage services.  I actually find it interesting that by default DocumentDB isn't encrypted at rest tbh.

Reply:
@patheard - This has been done. Now it is set to true by default and I updated the examples as well.